doc/source/contributing_docs.rst

```diff
@@ -83,6 +66,136 @@ Using ``conda`` or similar (``miniconda``, ``mamba``, ``micromamba``), create a
         conda activate mda-user-guide
         jupyter-nbextension enable nglview --py --sys-prefix
 
+
+
+.. _add-new-documentation:
+
+Adding new documentation
+========================
+
+The documentation is built using `Sphinx <https://www.sphinx-doc.org/en/master/>`_.
+The user guide is largely composed of four different kinds of files:
+
+* :ref:`reStructuredText files <restructuredtext-files>` (``.rst``) which contain the text of the documentation
+* :ref:`Jupyter notebooks <jupyter-files>` (``.ipynb``) which contain code examples
+* Python scripts (``.py``) in the ``doc/source/scripts/`` directory which are used
+  for automatically generated documentation
+* Text files (``.txt``) which contain the automatically generated documentation.
+  These should not be edited directly as changes will not be included.
+  These are largely tables and lists of topology attributes for readers, writers,
+  and parsers.
+
+
+What file to edit
+"""""""""""""""""
+The structure of the user guide is a bit convoluted!
```

Comment:
```suggestion
```

Reply:
I agree – never apologize for supplying documentation!